types/Account.ts

```diff
@@ -2,6 +2,7 @@ import { AccountValue } from '@ironfish/sdk'
 import AccountBalance from './AccountBalance'
 
 interface Account extends AccountValue {
```

Comment:
Not crazy about this boolean flag, but it seems like the best compromise with the current structure of these types